anomalib/data/utils/augmenter.py

```diff
@@ -36,9 +36,19 @@ class Augmenter:
     Args:
         anomaly_source_path (Optional[str]): Path to a folder of images that will be used as source of the anomalous
         noise. If not specified, random noise will be used instead.
+        p_anomalous (float): Probability that the anomalous perturbation will be applied to a given image.
```

Comment:
not related to this change, but I'm not sure if `Augmenter` would actually reveal what the class is doing. Maybe we could rename this class so everyone could easily understand what it does?

Reply:
I agree, but I also think this class needs some refactor in general. Would it be alright if we address this in a separate PR to speed things up?